docs/get-started/getting-started-alluxio.md

```diff
@@ -0,0 +1,200 @@
+   alluxio.user.ufs.block.read.location.policy=alluxio.client.block.policy.LocalFirstPolicy
+   ####################################
+
+   # Running Alluxio Locally with S3
+   alluxio.underfs.s3.endpoint=/S3_ENDPOINT
+   alluxio.underfs.s3.inherit.acl=false
+   alluxio.underfs.s3.default.mode=0755
+   alluxio.underfs.s3.disable.dns.buckets=true
+   ```
+
+   For the explanation of each configuration, please refer to [Alluxio Configuration](https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html) and [Amazon AWS S3](https://docs.alluxio.io/os/user/stable/en/ufs/S3.html).
+
+   Add Alluxio worker hostnames into `${ALLUXIO_HOME}/conf/workers`.
+
+   ``` json
+   NM_hostname_1
+   NM_hostname_2
+   ```
+
+   - Copy configuration from Alluxio master to Alluxio workers.
+
+   ``` shell
+   ${ALLUXIO_HOME}/bin/alluxio copyDir ${ALLUXIO_HOME}/conf
+   ```
+   - Alluxio worker configuration.
+
+   Add worker and user hostname respectively in the Alluxio configuration of each Alluxio worker.
+
+   ``` xml
+   alluxio.worker.hostname=NM_hostname_X
+   alluxio.user.hostname=NM_hostname_X
+   ```
+
+   Note that Alluxio can manage other storage media (e.g. MEM, HDD) in addition to ssd,
+   so local data access speed may vary depending on the local storage media. To learn
+   more about this topic, please refer to the [tiered storage document](https://docs.alluxio.io/os/user/stable/en/core-services/Caching.html#multiple-tier-storage).
+
+5. Mount an existing S3 bucket to Alluxio.
+
+   ``` bash
+   ${ALLUXIO_HOME}/bin/alluxio fs mount \
+      --option aws.accessKeyId=<AWS_ACCESS_KEY_ID> \
+      --option aws.secretKey=<AWS_SECRET_KEY_ID> \
+      alluxio://RM_hostname:19998/s3 s3a://<S3_BUCKET>/<S3_DIRECTORY>
+   ```
+
+   For other filesystem, please refer to [this site](https://www.alluxio.io/)
+
+6. Start Alluxio cluster.
+
+   Login Alluxio master node, and run
+
+   ``` bash
+   ${ALLUXIO_HOME}/bin/alluxio-start.sh all
+   ```
+
+## RAPIDS Configuration
+
+There are two ways to leverage Alluxio in RAPIDS.
+
+1. Explicitly specify alluxio path.
+
+   This may require user to change code.
+
+   Eg. Change below code
+
+   ``` scala
+   val df = spark.read.parquet("s3a://<S3_BUCKET>/<S3_DIRECTORY>/foo.parquet")
+   ```
+
+   to
+
+   ``` scala
+   val df = spark.read.parquet("alluxio://RM_hostname:19998/s3/foo.parquet")
+   ```
+
+2. Transparently replace in RAPIDS.
+
+   RAPIDS has added a configuration `spark.rapids.alluxio.pathsToReplace` which can allow RAPIDS
+   to replace the input file paths to alluxio paths transparently at runtime. So there is no any
+   code change for users.
+
+   Eg, at startup
+   ``` shell
+   --conf spark.rapids.alluxio.pathsToReplace="s3:/foo->alluxio://RM_hostname:19998/foo,gs:/bar->alluxio://RM_hostname:19998/bar"
+   ```
+
+   This configuration allows RAPIDS to replace any file paths prefixed `s3:/foo` to
+   `alluxio://RM_hostname:19998/foo` and `gs:/bar` to `alluxio://RM_hostname:19998/bar`
```

Comment:
we should state some of the caveats mentioned in code review.  input_file_name is going to print the alluxio:// path rather then the original. below is an example fo input_file_name.

```
lambda spark : spark.read.parquet(data_path)
.filter(f.col('a') > 0)
.selectExpr('a',
'input_file_name()',
'input_file_block_start()',
'input_file_block_length()'),
```